packages/database/supabase/functions/create-space/index.ts

```diff
@@ -0,0 +1,220 @@
+// Follow this setup guide to integrate the Deno language server with your editor:
+// https://deno.land/manual/getting_started/setup_your_environment
+// This enables autocomplete, go to definition, etc.
+
+// Setup type definitions for built-in Supabase Runtime APIs
+import "jsr:@supabase/functions-js/edge-runtime.d.ts";
+import {
+  createClient,
+  type User,
+  type PostgrestSingleResponse,
+  PostgrestError,
+} from "jsr:@supabase/supabase-js@2";
```

Comment:
Is there a way to make this type safe?
<img width="869" height="200" alt="image" src="https://github.com/user-attachments/assets/94182268-ee9c-4648-ada1-df39c3a8c233" />